e2e/README.md

```diff
@@ -37,3 +37,85 @@ If you have .env file, you can use CI or TEST_PROVIDER environment variables to
 - `TEST_PROVIDER=sauce npm run test:e2e`
 
 Both command do the same, use the saucelabs as a driver, but you get the results and the evidences locally.
+
+### pictures
+
+The results of the tests are written in the terminal, but you can check the created pictures about the elements in the same folder where the tests are located in the ./pics folder, next to the references.
+
+### clean the Tests
+
+If you have too many pics which is confusing you can run a command which clean every pics except of references.
+
+- `npm run clean-test`
+
+
+## Test developement guide
+
+The FEF tests based on visual testing. The basic concept is to take screenshot of every status of elements or pages and make difference to a reference picture.
+
+### Structure
+
+You can find everything regarding to end to end testing in the "e2e" folder. There are two important folders located in e2e: test and utils. The "test" contains the folders of the tests of the different pages. The "utils" contains the functions for the tests.
+
+### Test
+
+Every folder in the test folder contains a .js file and a pics folder. This is a whole page test, e.g. the [./e2e/test/select-tests](/e2e/test/select-tests/) folder contains every tests of [ux.Intellyo.com/select](http://ux.Intellyo.com/select).
+
+The tests located in the javascript file in groups.
+
+The references, screenshots and the diff pictures (created by the js file) located in the pics folder.
```

Comment:
~~The references, screenshots and the diff pictures (created by the js file) located in the pics folder.~~ 
⬇️
The **reference pictures**, screenshots and the diff pictures (created by the js file) **are**  located in the pics folder.